pkg/utils/util.go

```diff
@@ -390,3 +449,7 @@ func containsWithStringReturn(list []string, element string, fn func(string, str
 func containsNamespaceWithStringReturn(patterns []string, ns string) (string, string, bool) {
 	return containsWithStringReturn(patterns, ns, comparePatterns)
 }
+
+func SplitString(s string, sep string) []string {
```

Comment:
Is this used ?

Reply:
nah, i think it wasnt referenced after the cleanup

Reply:
Please remove it